syntax/Jenkinsfile.vim

```diff
@@ -76,7 +83,7 @@ syn keyword jenkinsfilePipelineStep klocworkQualityGateway klocworkWrapper kuber
 syn keyword jenkinsfilePipelineStep library libraryResource liquibaseDbDoc liquibaseRollback liquibaseUpdate listAWSAccounts
 syn keyword jenkinsfilePipelineStep livingDocs loadRunnerTest lock logstashSend mail marathon mattermostSend memoryMap
 syn keyword jenkinsfilePipelineStep milestone mockLoad newArtifactoryServer newBuildInfo newGradleBuild newMavenBuild
-syn keyword jenkinsfilePipelineStep nexusArtifactUploader nexusPolicyEvaluation nexusPublisher node nodejs nodesByLabel
+syn keyword jenkinsfilePipelineStep nexusArtifactUploader nexusPolicyEvaluation nexusPublisher nodejs nodesByLabel
```

Comment:
This is a judgment call.  Jenkins pipeline documentation seems to give special treatment to the `node` step, so it stays a `jenkinsfileCoreStep` above.